audio.go

```diff
@@ -13,29 +14,41 @@ const (
 	defaultHeightModifier float32 = 1
 )
 
+var MasterVolume float64 = 1
+
 // AudioComponent is a Component which is used by the AudioSystem
 type AudioComponent struct {
 	File       string
 	Repeat     bool
 	Background bool
 	player     *Player
+	RawVolume  float64
```

Comment:
You may want to note that `RawVolume` shouldn't be changed? Or maybe we want to make sure they don' change it?